doc/source/reference/array_api.rst

```diff
@@ -153,7 +153,9 @@ Keyword Argument Renames
 The following functions have keyword arguments that have been renamed. The
 functionality of the keyword argument is identical unless otherwise stated.
 Each new keyword argument is not already present on the given function in
```

Comment:
This is difficult to parse, could you rephrase it?

Reply:
I think I meant to delete this sentence in favor of the one I added.